acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeConfiguration.java

```diff
@@ -275,10 +271,6 @@ public Optional<KeyPair> getKeyPair() {
     return keyPair;
   }
 
-  public Optional<PkiKeyStoreConfiguration> getPkiKeyStoreConfiguration() {
-    return pkiKeyStoreConfiguration;
-  }
-
   public boolean isStrictTxReplayProtectionEnabled() {
```

Comment:
The p2pTLSEnabled method can be removed and also the create TLS methods in BesuNodeFactory